aws/resource_aws_budgets_budget_test.go

```diff
@@ -0,0 +1,408 @@
+package aws
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"strconv"
+	"testing"
+	"text/template"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/budgets"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSBudget_basic(t *testing.T) {
```

Comment:
These function names need to be updated as well, e.g. `TestAccAWSBudgetsBudget_basic`